tox.ini

```diff
@@ -15,6 +15,12 @@ commands=
         --capture=no \
         -v
 
+[testenv:lint]
```

Comment:
One more thing, flake8 and mccabe are run as part of the py testenv, is a separate linter env needed here? And wouldn't it have to be added to envlist in line 2?

Reply:
We tend to use pylint as part of our internal test process, so I added this section while I was working on the code for the familiar tool chain. I probably shouldn’t have pushed that part.

On the envlist issue... the envlist in the tox section is just the envs to be built by default. You can define additional envs but you have to specify them manually (e.g. tox -e lint)